flink-cdc-base/src/test/resources/ddl/inventory.sql

```diff
@@ -35,7 +35,8 @@ VALUES (default,"scooter","Small 2-wheel scooter",3.14),
        (default,"hammer","16oz carpenter's hammer",1.0),
        (default,"rocks","box of assorted rocks",5.3),
        (default,"jacket","water resistent black wind breaker",0.1),
-       (default,"spare tire","24 inch spare tire",22.2);
+       (default,"spare tire","24 inch spare tire",22.2),
+       (default,"spare tire","28 inch spare tire",26.2);
```

Comment:
Hi @molsionmo  , I need to add a new record, because id = 101 ~ 109 cannot be deleted. `products.id` is the foreign key of `products_on_hand` table.